src/components/artists-list.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { observer } from 'mobx-react'
+
+export default @observer class ArtistsList extends Component {
+  render() {
+    const { artistsStore } = this.props
+
+    return (
+      <div>
+        <ul className="artists-list__list">
```

Comment:
Trochę wydaje mi się zbędne powtarzanie list. Chyba lepiej wygląda `artists-list` i potem `artists-list__item` itp.

Reply:
na poczatku w ogole nie mialem artists-list__ ale pomyslalem, ze jak dam nazwe pliku to pozniej bedzie mi sie latwiej ogarnac jak bede podgladac w przegladarce co to za klasa mi tu problemy robi :p. Nazwe moze zmienie jak wpadne, bo juz list-item mam ponizej tez

Reply:
Ok, trochę zbędnie verbose, ale nie jest to błąd jako taki :P